src/Pages/Competition.jsx

```diff
@@ -1,22 +1,24 @@
 import { useParams } from "react-router-dom"
-import { useEffect } from "react"
+import { useEffect, useState } from "react"
 
-function Competition({setCompetitions}) {
+function Competition() {
     const { id } = useParams();
+    const [competition, setCompetition] = useState({})
 
     function getCompetition() {
         const url = `http://localhost:3030/competition/${id}`;
         fetch(url)
           .then((res) => res.json())
           .then((Data) => {
             console.log(Data)
-            setCompetitions(Data.data)
+            setCompetition(Data.data)
```

Comment:
We tend to not have our variables start with an uppercase in javascript

If these `console.log` calls are no longer used, it might be a nice idea to remove them so you get a cleaner log when running the code :)

Reply:
Fixed that just now